src/components/UI/Home_HeroSix.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../styleSheets/_variables.scss";
+@import "../../styleSheets/_mixins.scss";
+@import "./Home.scss";
+
+#heroSix {
+  @include flex(center, center, column);
+  @include background-image("/src/assets/images/landscapes/background-11.jpg");
```

Comment:
as a reminder we either need to fix the issue with images not loading from our package or swap out to imgur